### DISCUSSION TITLE
row.getValue(columnId) When Column with id  does not exist,  can I return null or undefined?

### DISCUSSION THREAD

**Discussion by MingRock** [TanStack/table#4360] [Ideas]
can I return null or undefined? 
Don't report an error

**Comment by KevinVandy** [comment]
Maybe in prod mode, but in development mode, it's good that it throws an error to prevent incorrect usage

**Reply by MingRock** [reply]
Can you add a Row API  :  row.getIsExist(ColumnId)   ?
